com.github.subhadeepjasu.pebbles.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.github.subhadeepjasu.pebbles
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+base: io.elementary.BaseApp
+base-version: juno-20.08
+command: com.github.subhadeepjasu.pebbles
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=session-bus
```

Comment:
```suggestion
```

Reply:
It won't work without it, please see https://github.com/flathub/flathub/pull/2263#discussion_r614365070 .

Reply:
It's discussed there that it will work now as app ID matches D-Bus ID.

Reply:
Ah okay, sorry for the misunderstanding.